src/test/java/com/faforever/client/fx/MouseEvents.java

```diff
@@ -11,4 +11,12 @@ private MouseEvents() {
   public static MouseEvent generateClick(MouseButton button, int clicks) {
     return new MouseEvent(MouseEvent.MOUSE_CLICKED, 0, 0, 0, 0, button, clicks, false, false, false, false, false, false, false, false, false, false, null);
   }
+
+  /**
+   * @param x the x with respect to the node.
+   * @param y the y with respect to the node.
+   */
+  public static MouseEvent generateMouseMoved(int x, int y) {
```

Comment:
FxRobot class does such things I think

Reply:
Cannot find the method I need. The `moveTo` method is not suitable because he moves the cursor physically. The `generateMouseMove` simply imitates the movement of the cursor to the node for invoking of `mouseMoved` listener.

Reply:
ok